pavelib/assets.py

```diff
@@ -35,6 +35,15 @@
 THEME_SASS_DIRECTORIES = []
 SASS_LOAD_PATHS = ['common/static', 'common/static/sass']
 
+# A list of NPM installed libraries that should be copied into the common
+# static directory.
+NPM_INSTALLED_LIBRARIES = [
+    'underscore/underscore.js'
+]
+
+# Directory to install static vendor files
+STATIC_VENDOR_DIRECTORY = 'common/static/common/js/vendor'
```

Comment:
I recall at some point the `openedx` was the new `common`?  Is there a reason to put this in `common` vs `openedx`?